### ISSUE TITLE
Bug: Hello World Error: Nats Authorization Violation

### ISSUE THREAD

**Issue by ColTrez** [memphisdev/memphis#952]
### Describe the bug

Memphis version: v1.0.2
Associated logs:
[pythonclient.txt](https://github.com/memphisdev/memphis/files/11619067/pythonclient.txt)
[dockerCompose.txt](https://github.com/memphisdev/memphis/files/11619069/dockerCompose.txt)

I'm trying to get memphis set up to explore it.
I've ran the stable docker compose, logged into the ui, set up a station and a user, and am trying to create a producer in Python. I'm using the code the tutorial in the ui suggests. The only change I've made to it is the producer_name, as the one given results in an error due to invalid characters. I get this error when I run it: nats.errors.Error: nats: 'Authorization Violation'.  The following code is used:

```from __future__ import annotations
import asyncio
from memphis import Memphis, Headers, MemphisError, MemphisConnectError, MemphisHeaderError, MemphisSchemaError


async def main():
    try:
        memphis = Memphis()
        await memphis.connect(host="localhost", username="cfm", password="memphis")

        producer = await memphis.producer(station_name="stationone",
                                          producer_name="producerone")  # you can send the message parameter as dict as well
        headers = Headers()
        headers.add("key", "value")
        for i in range(5):
            await producer.produce(bytearray("Message #" + str(i) + ": Hello world", "utf-8"), headers=headers)

    except (MemphisError, MemphisConnectError, MemphisHeaderError, MemphisSchemaError) as e:
        print(e)

    finally:
        await memphis.close()


if __name__ == "__main__":
    asyncio.run(main())
```

The following go sdk code appears to work without problem:

```
package main

import (
	"fmt"
	"github.com/memphisdev/memphis.go"
	"os"
)

func main() {
	conn, err := memphis.Connect("localhost", "cfm", memphis.Password("memphis"))
	if err != nil {
		os.Exit(1)
	}
	defer conn.Close()
	p, err := conn.CreateProducer("StationOne", "PRODUCER_NAME")

	hdrs := memphis.Headers{}
	hdrs.New()
	err = hdrs.Add("key", "value")

	if err != nil {
		fmt.Errorf("Header failed: %v", err)
		os.Exit(1)
	}

	err = p.Produce([]byte("You have a message!"), memphis.MsgHeaders(hdrs))

	if err != nil {
		fmt.Errorf("Produce failed: %v", err)
		os.Exit(1)
	}
}
```

### Steps to reproduce

1. Run the docker compose
2. Go to the ui, log in as root
3. Go through with the tutorial: create a station, and a user
4. In the Produce Data section of the tutorial, select python as the language
5. Install memphis-py
6. Attempt to run the code snippet given

### Affected services

SDKs

### Platforms

Docker

### Environment

Production

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by yanivbh1** [comment]
Thank you @ColTrez. It's under review and will be updated soon!

**Comment by idanasulinmemphis** [comment]
Hi @ColTrez thanks for bringing it to our attention. Basically, the error you are seeing this output is totally by mistake and functionality is totally fine. However, we've released a new version of the Python SDK (1.0.3) which fixes that issue
